frontend-public/src/components/dashboard/UserMineInfo.js

```diff
@@ -9,11 +10,11 @@ const propTypes = {
 
 // This file is anticipated to use state
 // eslint-disable-next-line react/prefer-stateless-function
-export class ProponentDashboard extends Component {
+export class UserMineInfo extends Component {
```

Comment:
I find this naming a bit confusing. Should this just be MineInfo maybe?

Reply:
I think that was meant to be UserDashboard

Reply:
Sorry I was just confused by the proponentDashboard above it and MineInfo would be good

Reply:
All renamed.